flake.nix

```diff
@@ -58,6 +58,7 @@
             "teku" = inputs.ethereum-nix.packages.${system}.teku;
             # Main
             "backend" = pkgs.callPackage ./packages/backend { };
+            "frontend" = pkgs.callPackage ./webui { };
```

Comment:
Can this be moved under the packages dir?

Reply:
Yes, but let us do that in a separate chores PR.